packages/core/content-type-builder/admin/src/components/FormModal/index.js

```diff
@@ -955,29 +956,51 @@ const FormModal = () => {
                 }}
               >
                 <Flex justifyContent="space-between">
-                  <Typography as="h2" variant="beta">
-                    {formatMessage(
-                      {
-                        id: getModalTitleSubHeader({
-                          actionType,
-                          forTarget,
-                          kind,
+                  {customField ? (
```

Comment:
I think this block should be a component, similar to what Rémi did here: https://github.com/strapi/strapi/commit/0ebee64dd3c6d19470c60b6e44b36f145ff33919#diff-0b8e8963b7fe194c286e2a094a5d3ca33ff1b65afb86d81865c39d6af95cf05f

Reply:
I think you should use `modalType === 'customField'` for the condition when you move to a component